TsRandomizer.ItemTracker/TrackerRenderer.cs

```diff
@@ -92,6 +92,7 @@ public void Draw(SpriteBatch spriteBatch, ItemTrackerState state)
 				DrawPinkSource(spriteBatch, state);
 				DrawLaserAccess(spriteBatch, state);
 				DrawLabAccess(spriteBatch, state);
+				DrawItem(spriteBatch, state.DrawbridgeKey, 236, Color.White);
```

Comment:
There didn't seem to be a better way to get the correct (temporary?) icon for the item to be accessed by way of an item type, so instead this goes via the animation index and a bit of refactoring.